testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/http/SpnegoMechTestCase.java

```diff
@@ -174,49 +163,33 @@ public void testSuccess() throws Exception {
     /**
      * A setup task which configures and starts Kerberos KDC server.
      */
-    @CreateDS(
-            name = "JBossDS-SpnegoMechTestCase",
-            factory = org.jboss.as.test.integration.ldap.InMemoryDirectoryServiceFactory.class,
-            partitions = {
-                    @CreatePartition(
-                            name = "wildfly",
-                            suffix = "dc=wildfly,dc=org",
-                            contextEntry = @ContextEntry(
-                                    entryLdif = "dn: dc=wildfly,dc=org\n" +
-                                                "dc: wildfly\n" +
-                                                "objectClass: top\n" +
-                                                "objectClass: domain\n\n"))
-            },
-            additionalInterceptors = { KeyDerivationInterceptor.class })
-    @CreateKdcServer(
-            primaryRealm = "WILDFLY.ORG",
-            kdcPrincipal = "krbtgt/WILDFLY.ORG@WILDFLY.ORG",
```

Comment:
I don't see "krbtgt/WILDFLY.ORG@WILDFLY.ORG" referenced below in the new changes. I guess the tests weren't relying on this?

Reply:
@fjuma It is created in simpleKdcServer.init(); , same file, line 180